CRM/Civicase/Hook/BuildForm/TokenTree.php

```diff
@@ -179,6 +189,24 @@ private function addCaseRoleTokens(array $caseRoleTokens, array &$newTokenTree)
     $this->processAndAddCaseRoleTokens($contactRoleTokens, $newTokenTree);
   }
 
+  /**
+   * Add case tokens to the new token tree.
```

Comment:
This description does not match the function name. Can you add a proper description?